README.md

```diff
@@ -253,18 +253,6 @@ git push --set-upstream <branch-name>
 After pushing the new branch, browse to your fork of the repository in GitHub and create a pull request from there.
 Once the pull request is created, please make changes to your branch based on the comments & discussion in the PR.
 
-## Releasing for beta versions
-
-Current release process:
-
-* Update CHANGELOG.md
-* Test GoReleaser config with `goreleaser check`
-* Tag a commit with the version you want to release e.g. `v1.2.3`
-* Push the tag & commit to GitHub
-  * GitHub actions will automatically set the version based on the tag, create a GitHub release, build the project, and upload binaries & SHA sum to the GitHub release
-* [Edit the new release in GitHub](https://github.com/UpCloudLtd/upcloud-cli/releases) and add the changelog for this release
-* Done!
-
```

Comment:
Do we have this documented somewhere else? If not, we could just update this to `## Releasing`. I have not usually done that `goreleaser check` step, but otherwise this seems up-to-date.